src/pages/cases/Case.tsx

```diff
@@ -27,6 +28,8 @@ const Case: React.FC<Props> = ({ match }) => {
     title: '', snippet: '', facts_and_issues: '', right_and_principle: '', interpretation: '',
   });
   const [topics, setTopics] = useState([]);
+  const [onSearch, setOnSearch] = useState(false);
+  const doc = document.querySelector('.main-case-content') as HTMLElement;
```

Comment:
This works by luck, because it actually gets its value BEFORE the div gets rendered! The way you've done it elsewhere -- with the ref -- is the best way.